Before: f2920268a60bd90c5d24222d7ad0c443c65736254d43e7b87c0f89aa81f3cfb7
After: ca7036c625bb942c3cdd0fc63f94a3500340a8b5d930024ca4641afec61e42c1
Patch: d26a0b37ef190443560847304716872af4d98ad60d84d555fc761d915e410279
--- lib/corvid/%name%_plugin.rb.tt	2012-08-13 12:56:43.053579014 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,10 +0,0 @@
-require 'corvid/plugin'
-
-class <%= class_name %> < Corvid::Plugin
-
-  require_path '<%= require_path %>'
-
-  feature_manifest ({
-  })
-
-end
--- test/spec/%name%_plugin_spec.rb.tt	2012-08-13 12:56:41.183568357 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,11 +0,0 @@
-# encoding: utf-8
-require_relative '../bootstrap/spec'
-require 'corvid/test/resource_patch_tests'
-require '<%= require_path %>'
-
-describe <%= class_name %> do
-  include Corvid::ResourcePatchTests
-  res_patch_dir "#{APP_ROOT}/resources"
-
-  include_feature_update_install_tests <%= class_name %>.new
-end
